_posts/2020-09-22-having-a-devops-dream.md

```diff
@@ -9,6 +9,10 @@ image: img/devops-dream/dds_mobile.jpg
 
 <centre><a href="https://devops.games"><img src="/img/devops-dream/dds_mobile.jpg" alt="A person in their workplace playing DevOps Dream on a mobile phone." /></a></centre>
 
+This post is part of a series covering our development of DevOps Dream:
+1. Having a DevOps Dream
+1. [Living the DevOps Dream][3]
+
 <br/>
 In case you hadn't heard, we did a thing! We released [DevOps Dream](https://dream.devops.games/) on 19th August 2020.  I'd like to share our motivations and give some insights behind the scenes.  Along the way, you might just pick up some tips on how you can Live the Dream and beat the game! ;o)
```

Comment:
Should you link to devops.games rather than directly to the game?

Reply:
Good point.